views/proxy_confirmation.pug

```diff
@@ -0,0 +1,14 @@
+extends layout
+
+block content
+  section.divider#request-info
+    h3= _("Oauth request for %s from: %s").format(kind, redirect_address)
+    div.panel-body
+          p.text-center
+            != _('Are you sure you would like to grant <a href="%s" target="_blank"> %s </a> access to the credentials of device: %s').format(redirect_address, redirect_address, kind)
+    div.container
+
+      p.text-center
+        a.btn.btn-primary(href='proxy/oauth2/%s'.format(kind))= _("Continue")
```

Comment:
This needs to be a form and a POST, to avoid a Cross Site Request Forgery attack where a malicious website skips the confirmation page.